Classes/Middleware/StyleguideRouter.php

```diff
@@ -134,7 +138,7 @@ public function process(
 
             if ($styleguideLanguage) {
                 // Set language in TSFE object
-                $GLOBALS['TSFE']->lang = $styleguideLanguage['identifier'];
```

Comment:
this needs to be reviewed: I am not sure if this is still a leftover of TYPO3 10.4